### PR TITLE
[Exporter.Geneva] Add support for UpDownCounter and ObservableUpDownCounter

### PR DESCRIPTION
**Pull request by utpilla**
## Changes
- Add support for UpDownCounter and ObservableUpDownCounter

For significant contributions please make sure you have completed the following items:

* [x] Appropriate `CHANGELOG.md` updated for non-trivial changes

### CODE REVIEW

**Review by mic-max — APPROVED** [review]

**Review by Kielek — APPROVED** [review]

File: src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs [hunk 1]
```diff
@@ -121,6 +121,7 @@ public override ExportResult Export(in Batch<Metric> batch)
                     switch (metric.MetricType)
                     {
                         case MetricType.LongSum:
+                        case MetricType.LongSumNonMonotonic:
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs:124]: need to handle -ve values?

**Review by cijothomas — APPROVED** [review]
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/685/files#r994145083
Need to confirm how to deal with negative values.

**Review by cijothomas — CHANGES_REQUESTED** [review]
Approved by mistake. see previous comment.

File: src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs [hunk 2]
```diff
@@ -148,7 +148,25 @@ public override ExportResult Export(in Batch<Metric> batch)
                                 break;
                             }
 
+                        // The value here could be negative hence we have to use `MetricEventType.DoubleMetric`
+                        case MetricType.LongSumNonMonotonic:
+                            {
+                                // potential for minor precision loss implicitly going from long->double
+                                // see: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/numeric-conversions#implicit-numeric-conversions
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs:155]:
```suggestion
                                // see: https://docs.microsoft.com/dotnet/csharp/language-reference/builtin-types/numeric-conversions#implicit-numeric-conversions
```

**Review by cijothomas — APPROVED** [review]

**Pull request merged**